src/eq_schema/schema/Questionnaire/index.test.js

```diff
@@ -1,5 +1,5 @@
 const { BUSINESS } = require("../../../constants/questionnaireTypes");
-const { DEFAULT_METADATA } = require("../../../constants/metadata");
+// const { DEFAULT_METADATA } = require("../../../constants/metadata");
```

Comment:
Line should be removed rather than commented as what is refers to has been deleted